speedy-recovery-client/src/service/BackendMappingExtractionUtils.js

```diff
@@ -10,6 +10,21 @@ const getPosition = (messageResource, id) => {
   return "right";
 };
 
+const getMessageTitle = (position,role,name,title) =>{
+  if(role === "Practitioner") {

```

Comment:
This could be made a bit tidier like this:

`if (position === 'right') {   
return name;  
}
role === 'Practitioner' ? 'Parent' : title;`